qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteringRule.java

```diff
@@ -258,15 +258,17 @@ protected void before() throws IOException {
   @Override
   protected void after() {
     LOG.debug("Closing ClusteringRule...");
-    brokers.values().parallelStream()
```

Comment:
I guess it makes sense to add a comment here that we have changed it from parallelStream.

BTW I checked the implementation and have no clear indication why it did;t worked. Might be related to the values returned by the HashMap? :thinking: How have you experienced that?

Reply:
> BTW I checked the implementation and have no clear indication why it did;t worked

Yeah, this is not clear to me too but I speculated a bit in the issue description:

> either because the value collection couldn't be transformed to a parallel stream or maybe because there's some threshold for sized streams that is used to determine if the stream should be processed in parallel or not

> How have you experienced that?

I was looking at a test's log because I was annoyed that it takes a long time to run it. In the logs I noticed that brokers were apparently stopped sequentially.

Reply:
> I guess it makes sense to add a comment here that we have changed it from parallelStream

I'll add some more details in the commit message, IMO it makes no sense to add a code comment why no longer existing code that was used before doesn't work.

Reply:
weird :roll_eyes:

Reply:
I mean it is possible that it returns an sequenced one `It is allowable for this method to return a sequential stream.` based on the docs https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html#parallelStream-- 

But with the debug I was sure to see a parallel one :shrug: